modules/vba_documents/spec/lib/multipart_parser_spec.rb

```diff
@@ -9,127 +9,145 @@
   include VBADocuments::Fixtures
 
   describe '#parse' do
-    module FixtureHelper
-      class << self
-        attr_reader :data_type
-
-        def fetch(fixture, type)
-          r_val = fixture.path
-          r_val = StringIO.new File.binread(r_val) if type.eql? :stringio
-          r_val
-        end
```

Comment:
We no longer need this fixture helper because we are only handling `File`s from now on (no more `StringIO`s).

This also allowed me to put all test assertions at the root of `describe '#parse'` (see changes below).